src/random.ts

```diff
@@ -608,4 +608,27 @@ export class Random {
 
     return this.faker.datatype.hexadecimal(count);
   }
+
+  /**
+   * Returns the result of the callback if the probability check was successful, otherwise `undefined`.
+   *
+   * @param callback The callback to that will be invoked if the probability check was successful.
+   * @param options The options to use. Defaults to `{}`.
+   * @param options.probability The probability of the callback being invoked. Defaults to `0.5`.
```

Comment:
```suggestion
   * @param options.probability The probability (`[0.00, 1.00]`) of the callback being invoked. Defaults to `0.5`.
```

Maybe write a hint that the precision is capped at 2 decimal places?

Reply:
It's because of the default precision of `float`?

Reply:
Yes, for the boolean probability version I thought about deriving the precision from the inputs.